src/devices/Common/Iot/Device/Graphics/BdfFont.cs

```diff
@@ -0,0 +1,266 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Iot.Device.Graphics
+{
+    public class BdfFont
+    {
+        public int Width  { private set; get; }
+        public int Height { private set; get; }
+        public int XDisplacement { private set; get; }
+        public int YDisplacement { private set; get; }
+        public int DefaultChar  { private set; get; }
+        public int CharsCount   { private set; get; }
+
+        private Dictionary<int, int> GlyphMapper { set; get; }
+        private int BytesPerGlyph { set; get; }
+        private ushort[] GlyphUshortData { set; get; }
+
+        private static readonly string s_fontBoundingBox    = "FONTBOUNDINGBOX ";
+        private static readonly string s_charSet            = "CHARSET_REGISTRY ";
+        private static readonly string s_isoCharset         = "\"ISO10646\"";
+        private static readonly string s_defaultChar        = "DEFAULT_CHAR ";
+        private static readonly string s_Chars              = "CHARS ";
+        private static readonly string s_startChar          = "STARTCHAR ";
+        private static readonly string s_encoding           = "ENCODING ";
+        // private static readonly string s_sWidth             = "SWIDTH";
```

Comment:
why is this commented out?

Reply:
because we don't support all font properties. I believe in the future we should enhance this class.